src/lib/converter/types/intrinsic.ts

```diff
@@ -4,6 +4,20 @@ import {Type, IntrinsicType} from "../../models/index";
 import {Component, ConverterTypeComponent, ITypeTypeConverter} from "../components";
 import {Context} from "../context";
 
+// Copy typescript's @internal enum set from:
+// https://github.com/Microsoft/TypeScript/blob/v2.0.5/src/compiler/types.ts#L2297-L2298
```

Comment:
The code in `ts-internal.ts` exposes the existing internal/hidden APIs by augmenting the "typescript" module and exported interfaces.

In this case, the `TypeFlags` enum itself is exported from `ts` namespace.  However, the `Intrinsic` enum value is marked as `@hidden`.  I could not find a way to augment the  existing `TypeFlags` enum (which is exposed as a public API) with a new enum value (which is @internal and hidden).   Since I couldn't cleanly expose something from the `typescript` module using an augmentation, I decided to move it outside that file.  

The other option, I suppose, is to export it from the `ts-internal.ts` file as a separate symbol (not inside the `typescript` namespace).  I don't really have a strong opinion on this.